src/helpers.py

```diff
@@ -22,4 +22,6 @@ def sh(cmd: str, workdir: Optional[str] = None) -> int:
     if workdir is not None:
         msg += f" in '{workdir}'"
     substep(msg)
-    return subprocess.call(cmd, shell=True, cwd=workdir, executable="/bin/bash")
+    return subprocess.call(
+        f"set -euo pipefail; {cmd}", shell=True, cwd=workdir, executable="/bin/bash"
```

Comment:
Without this (`-e`) something like `_check_sh("false; true")` would exit with `0`, which we do *not* want. Also without `-o pipefail` something like `_check_sh("false | true")` exits with `0`, which we also don't want.